content/docs/user-guide/interactivity/user-interface/slices/working-slices-newtab.md

```diff
@@ -11,7 +13,7 @@ When you edit an instance of a UI slice in a UI canvas, you can use the context
 
 1. Choose **Edit slice in new tab** and then choose the slice you want to edit. Multiple choices appear only if the selected element is an instance of a cascaded slice.
 
-![Right-click the slice element to display the context menu.](/images/user-guide/game_ui_editor/ui-editor-working-slices-newtab.png)
+![Right-click the slice element to display the context menu.](/images/user-guide/interactivity/user-interface/slices/ui-editor-working-slices-newtab.png)
```

Comment:
The ordered list numbering resets after this image. To fix this, you'll need to indent this image.